src/System.CommandLine.DragonFruit.Tests/ConfigureFromMethodTests.cs

```diff
@@ -221,6 +221,19 @@ public void Options_are_not_built_for_infrastructure_types_exposed_by_method_par
                    .NotContain(o => o.Argument.ArgumentType == type);
         }
 
+        [Fact]
+        public void Parameters_with_Argument_suffix_is_treated_as_required_arguments()
```

Comment:
Whether or not the argument is required depends on its arity. But this is also orthogonal to the naming convention, so I'd rename the test to remove "required".